app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -297,15 +301,7 @@ class EventDetailsFragment : Fragment() {
         Picasso.get()
             .load(event.originalImageUrl)
             .placeholder(R.drawable.header)
-            .into(rootView.eventImage, object : Callback {
-                override fun onSuccess() {
-                    startPostponedEnterTransition()
-                }
-
-                override fun onError(e: Exception?) {
-                    startPostponedEnterTransition()
-                }
-            })
+            .into(rootView.eventImage)
```

Comment:
Can we pass the event image from Events fragment to event details fragment, so there will be no delay for loading image using URL and Picasso?